app/main.ipynb

```diff
@@ -678,16 +971,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 35,
-   "outputs": [],
+   "execution_count": 127,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "array([ 3, 13, 11, 24, 21])"
+     },
+     "execution_count": 127,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# write your numpy code here"
+    "my_list = np.array([6, 9, 12, 14, 17, 20])\n",
+    "mask = (my_list % 2 == 0) & (my_list % 3 == 0) | (my_list > 10)\n",
+    "np.bitwise_xor(my_list[mask], 7) + 2"
```

Comment:
incorrect result
the issue is with the order of execution binary operations
original: if num % 2 == 0 and (num % 3 == 0 or num > 10)
if you remove parenthesises(this is what you did), it'll look like
if num % 2 == 0 and num % 3 == 0 or num > 10
the execution ordering of such operators is: not, and, or
in your case and will be executed first with num % 2 == 0 and num % 3 == 0 and then or condition
if you add parenthesises, then "if num % 2 == 0 and (num % 3 == 0 or num > 10)"
"and" will be executed between the first condition and the result of "or" operation